table/block_based/filter_block.h

```diff
@@ -75,12 +78,28 @@ class FilterBlockBuilder {
   }
   // If filter_data is not nullptr, Finish() may transfer ownership of
   // underlying filter data to the caller,  so that it can be freed as soon as
-  // possible.
+  // possible. BlockBasedFilterBlock will ignore this parameter.
+  //
+  // If filter_data_cache_res_handle is null, Finish() may transfer
+  // ownership of cache reservation for the memory used by filter data
```

Comment:
Todo:
“.. ownership of cache reservation when reserving feature is available …”

Reply:
No longer needed due to API change